src/vsphere_cpi/lib/cloud/vsphere/resources/network.rb

```diff
@@ -0,0 +1,93 @@
+require 'cloud/vsphere/resources'
+
+module VSphereCloud
+  module Resources
+    class Network
+      include VimSdk
+      include ObjectStringifier
+      stringify_with :name, :mob, :bosh_name
+
+      attr_reader :mob, :client, :bosh_name, :vim_name
+
+      def self.NetworkBuilder(bosh_name, mob, client)
```

Comment:
I think this can just be called `make` or something like that.

Reply:
Done